background/constants/networks.ts

```diff
@@ -433,7 +236,7 @@ export const DEFAULT_QUAI_LOCAL = {
       shard: "cyprus-1",
       rpc: "http://localhost:8610",
       blockExplorerUrl: "https://dev.cyprus1.quaiscan.io",
-      multicall: "0x15b6351eDEcd7142ac4c6fE54948b603D4566862"
+      multicall: "0x1aF3f0d55e6f399708a058333E81604F56f22835"//"0x15b6351eDEcd7142ac4c6fE54948b603D4566862"
```

Comment:
Dead comment?

Reply:
Technically none of the 'multicall' addresses are correct because they have to be manually deployed on the chain. But I've removed the comment.